src/cpp/rtps/builtin/discovery/participant/PDPClient.cpp

```diff
@@ -91,6 +91,10 @@ void PDPClient::initializeParticipantProxyData(
         participant_data->m_availableBuiltinEndpoints |= DISC_BUILTIN_ENDPOINT_SUBSCRIPTION_ANNOUNCER;
     }
 
+    // Set participant type and discovery server version properties
+    participant_data->m_properties.push_back(std::pair<std::string, std::string>({"PARTICIPANT_TYPE", "CLIENT"}));
+    participant_data->m_properties.push_back(std::pair<std::string, std::string>({"DS_VERSION", "2.0"}));
```

Comment:
This constant should be stored as a macro.

Reply:
I've made some constants